app/services/api/v3/ecf/participants_query.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+module Api
+  module V3
+    module ECF
+      class ParticipantsQuery
+        def initialize(lead_provider:, params:)
+          @lead_provider = lead_provider
+          @params = params
+        end
+
+        def participants
+          join = InductionRecord
+                   .select("DISTINCT FIRST_VALUE(induction_records.id) OVER (#{latest_induction_record_order}) AS latest_id, induction_records.training_status")
+                   .joins(:participant_profile, :schedule, { induction_programme: :partnership })
+                   .where(
+                     schedule: { cohort_id: with_cohorts.map(&:id) },
+                     induction_programme: {
+                       partnerships: {
+                         lead_provider_id: lead_provider.id,
+                         challenged_at: nil,
+                         challenge_reason: nil,
+                         pending: false,
+                       },
+                     },
+                   )
+
+          scope = User
```

Comment:
could we add the selects back please as they reduced the query times? (can reuse a lot from v1)?